lib/config/ChainConfigManager.ts

```diff
@@ -9,7 +9,7 @@ type IntentOverrides = {
 };
 
 export type DutchOverrides = IntentOverrides & {
-  skipRFQ?: boolean;
+  forceOpenOrders?: boolean;
```

Comment:
I doubt it but maybe good to check if `skipRFQ` is used by anyone?

Reply:
I just added it last week. It's self-contained in the URA service.